Makefile

```diff
@@ -44,7 +44,7 @@ unit-test:
 .PHONY: wallet-web-test
 wallet-web-test:
 	@set -e
-	@cd cmd/wallet-web && npm install && npm run test
+	# @cd cmd/wallet-web && npm install && npm run test
```

Comment:
is this by mistake or required for nnow ??

Reply:
@talwinder50 - for now we will disable the unit tests and merge this PR - (discussed this with @sudeshrshetty and @birtony)

Reply:
Opened #1234 to address this @talwinder50 @rolsonquadras